libcontainer/criurpc/criurpc.proto

```diff
@@ -1,3 +1,5 @@
+syntax = "proto2";
```

Comment:
No, we can't, because criu doesn't support proto3. It is a part of CRIU API: https://github.com/xemul/criu/blob/master/images/rpc.proto